app/components/document_metadata_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <div class="item-provider col-lg-8 offset-lg-2">
```

Comment:
![Screenshot 2023-02-21 at 12 05 41](https://user-images.githubusercontent.com/4924494/220447120-3f824114-d8b8-4b2e-81b2-d17928389b32.png)

this ended up looking cramped for me on most decent-size screens. i think `col-lg-10 offset-lg-1` looked better:

![Screenshot 2023-02-21 at 12 06 22](https://user-images.githubusercontent.com/4924494/220447253-95b4cbb3-1a73-4d5a-8187-b6ca20cf88b2.png)

Reply:
That seems unbalanced to me.  Almost all of the text is left of center, leaving a bunch of white space on the right.

Reply:
@thatbudakguy the field labels come from whatever is configured in the exhibit dashboard, and in prod they've been updated to: "Holding institution" and "Data provider." So I don't think there's an issue with the labels and this PR.

Reply:
Ah, so it's user-defined. That makes sense

Reply:
Re: the sizing of the box, could we just decrease the padding or something? The wrapping that's happening now seemed to be what Gary was explicitly trying to avoid

Reply:
It's not padding, it's how Bootstrap defines balanced definition lists, which is the proper semantic markup.

Reply:
I'm talking about the padding of the box. In Gary's design it looks smaller than in the PR, which gives the content more space. His design is more similar to the second screenshot I posted.

I don't think we should ship something where the text is wrapping on common screen sizes.

<img width="1039" alt="149239215-6b0cdccc-b260-41af-8704-ce3f46972e84" src="https://user-images.githubusercontent.com/4924494/220728675-0567d44d-431c-4e1d-9d61-071ecdf8c38b.png">

Reply:
@thatbudakguy I don't think definition lists work that way in bootstrap: https://getbootstrap.com/docs/4.0/content/typography/#description-list-alignment

Reply:
Here's what it looks like with the padding on `.item-provider` (which is our style) decreased to 0.5rem, and switching the column split to 4/8 instead of 3/9 so that the `<dt>` can fit. That way, if text wraps, it's the `<dd>` that are wrapping. I think this is closer to Gary's design.

![Screenshot 2023-02-22 at 11 56 06](https://user-images.githubusercontent.com/4924494/220744357-67a3bf24-82bb-4369-9497-cac6edd53cfe.png)